### ISSUE TITLE
Gujarati - ન - ત Confusion

### ISSUE THREAD

**Issue by Shreeshrii** [tesseract-ocr/tessdata#60]
When using tesseract 4.0 with --oem 1 (LSTM) with Gujarati traineddata, ન is being recognized as ત in the attached image. 

Same image when recognized with --oem 0 is recognizing ન correctly, but has other accuracy problems.

So, it looks like that LSTM model for Gujarati has not been trained with this font.

Image and ground truth file are attached.

It would be helpful to have the ability to finetune  using real images in addition to synthetic data.

[guj.ag.exp0-GT.txt](https://github.com/tesseract-ocr/tessdata/files/1098158/guj.ag.exp0-GT.txt)
![guj ag exp0](https://user-images.githubusercontent.com/5095331/27487513-932ae79a-5851-11e7-8d25-ef6b1a93aa53.png)

**Comment by Shreeshrii** [comment]
Tested with both best/guj and best/Gujarati traineddata - psm 6 just now.

While the ન - ત Confusion is still there, Gujarati traineddata seems better than guj - it is dropping fewer words in OCR output.

**Comment by supersaiyan12** [comment]
Hi @Shreeshrii 
I have number of images and wanted to convert into a text using tesseract 4.0.(All are in hand-written english)
_I have 2 query regarding this._
1.Can i make my own **".traineddata"** from the images that i have.
2.If yes, then how do i get start from the scratch.

Thanks in Advance
